openedx/frontend-app-course-authoring
# fix: sort of boolean columns

JIRA Ticket: [TNL-11221](https://2u-internal.atlassian.net/browse/TNL-11221)

>On Transcript and Active, clicking the toggle seemed to randomly resort items but not based on expectation. I would expect Active to sort between those with checkmarks and those without. I would expect Transcript to do same (agnostic as to whether number of transcripts needs to be a factor but couldn't hurt).

Testing

1. Navigate to the the video page
2. Switch to the list view
3. Click on the transcript table header
4. All videos with transcripts should move to the top
5. Click on the transcript header again
6. All videos without transcripts should move to the top
7. Click on the active table header
8. All the videos that are used in the course should move to the top
9. Click on the active table header again
10. All the videos that are not used in the course should move to the top
11. Navigate to the files page
12. Switch to the list view
13. Click on the locked table header
14. All locked files should move to the top
15. Click on the locked header again
16. All public (not locked) files should move to the top
17. Click on the active table header
18. All the videos that are used in the course should move to the top
19. Click on the active table header again
20. All the videos that are not used in the course should move to the top
